spec/models/conference_spec.rb

```diff
@@ -1656,4 +1656,23 @@
       expect{ conference.save }.to change{ EventSchedule.count }.from(2).to(1)
     end
   end
+
+  describe '#upcoming_conferences', blah: true do
+    let!(:upcoming_conference) { create(:conference) }
+    let!(:past_conf) { create(:conference, start_date: Date.current - 1.days, end_date: Date.current - 1.days) }
+
+    it 'should return correct count of upcoming conferences' do
+      expect(Conference.upcoming_conferences).to eq [upcoming_conference]
+    end
```

Comment:
So you can also add `subject { Conference.upcoming }` above the `let!` calls and then your test can look like:

```ruby
it { is_expected.to eq [upcoming_conference] }
```